src/scala/com/github/johnynek/bazel_deps/IO.scala

```diff
@@ -107,8 +119,14 @@ object IO {
       case WriteFile(f, d) =>
         Try {
           val os = new FileOutputStream(fileFor(f))
-          try os.write(d.value.getBytes("UTF-8")) finally { os.close() }
+          try os.write(d.value.getBytes(charset)) finally { os.close() }
         }
+      case Compare(f, d) => Try({
+        val ff = fileFor(f)
+        // FIXME this data comparison is inefficient
```

Comment:
I think this is likely fine for the sizes we expect here right? It might even be faster than some fancier methods since `Arrays.equals` is fast, no?

Reply:
Yeah, it's likely fast enough for the kilobytes-ish files we're dealing with.  I'm happy to not get fancier.